bin/local/create-local-user.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+set -e
+
+### failsafe logic to exit in case we are not running in our local minikube context
+KUBE_CONTEXT=$(kubectl config current-context)
+
+echo "Current kube context: '${KUBE_CONTEXT}'"
+
+if [[ "${KUBE_CONTEXT}" != "minikube-wbaas" ]]; then
+    echo "Error: wrong kube context. Use this script only within 'minikube-wbaas'!"
+    exit 1
+fi
+#####################################################################################
+
+# This script should create and verify a user account on a local wbaas cluster with the credentials below. Do not use this in production! 
+
+USER_CODE="${USER_CODE:-create-local-user}"
+USER_MAIL="${USER_MAIL:-jane.doe@wikimedia.de}"
+USER_PASS="${USER_PASS:-wikiwikiwiki}"
```

Comment:
Seems like GitGuardian is flagging the default password again and again, maybe we should remove it?

Reply:
Without ever having used it, I would guess it's possible to tell GitGuardian this is an ok case?

Reply:
I marked it as a test credential, let's see if it keeps that in mind in the future https://dashboard.gitguardian.com/workspace/306574/incidents/6671167?sort_date=true